### PR TITLE
Variable

### PR DESCRIPTION
**Pull request by kunyuan**
1. Add generic variable: Continuous
2. Add tutorial in readme.md
3. Initialize variables with different values #3 (FermiK has not yet been fixed. But it is less dangerous than Tau variables)
4. Check the corner case when normalization is zero or not a number.

### CODE REVIEW

File: src/variable.jl [hunk 1]
```diff
@@ -205,8 +207,8 @@ mutable struct RadialFermiK <: Variable
     data::Vector{Float64}
     kF::Float64
     δk::Float64
-    function RadialFermiK(kF=1.0, δk=0.01, size=MaxOrder)
-        k = [kF for i = 1:size]
+    function RadialFermiK(kF = 1.0, δk = 0.01, size = MaxOrder)
+        k = [kF * (i - 0.5) / size for i = 1:size] #avoid dulication
```
Comment (iintSjds) [inline src/variable.jl:211]: duplication?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,95 @@
...
 [![Stable](https://img.shields.io/badge/docs-stable-blue.svg)](https://numericalEFT.github.io/MCIntegration.jl/stable)
 [![Dev](https://img.shields.io/badge/docs-dev-blue.svg)](https://numericalEFT.github.io/MCIntegration.jl/dev)
 [![Build Status](https://github.com/numericalEFT/MCIntegration.jl/workflows/CI/badge.svg)](https://github.com/numericalEFT/MCIntegration.jl/actions)
 [![Coverage](https://codecov.io/gh/numericalEFT/MCIntegration.jl/branch/master/graph/badge.svg)](https://codecov.io/gh/numericalEFT/MCIntegration.jl)
+
+MCIntegration is Monte Carlo calculator for generic high-dimensional integral. The internal algorithm and some simple benchmarks can be found in the [document](docs/src/man/important_sampling.md).
+
+# Quick start
+
+The following example demonstrates the basic usage of this package. This code calculates the are of a circle and the volume of a sphere using one Markov chain. The code can be found [here](example/sphere.jl).
```
Comment (iintSjds) [inline README.md:14]: the area of a circle

File: src/variable.jl [hunk 3]
```diff
@@ -220,17 +222,36 @@ mutable struct Tau <: Variable
...
+mutable struct Continuous <: Variable
+    data::Vector{Float64}
+    λ::Float64
+    lower::Float64
+    range::Float64
+    function Continuous(bound, λ = nothing, size = MaxOrder)
+        lower, upper = bound
+        @assert upper > lower
+        @assert isnothing(λ) || (0 < λ < (upper - lower))
+        t = [lower + (upper - lower) * (i - 0.5) / size for i = 1:size] #avoid dulication
```
Comment (iintSjds) [inline src/variable.jl:240]: why is t initialized here this way while in sampler.jl the create!() initialized the variable randomly?

File: src/variable.jl [hunk 4]
```diff
@@ -130,17 +130,19 @@ mutable struct Configuration{V,P,O}
         if neighbor == []
             # By default, only the order-1 and order+1 diagrams are considered to be the neighbors
             # Nd is the normalization diagram, by default, it only connects to the first diagram
-            neighbor = Vector{Vector{Int}}([[d - 1, d + 1] for d in 1:Nd])
-            neighbor[1] = (Nd == 2 ? [2, ] : [Nd, 2]) # if Nd=2, then 2 must be the normalization diagram
-            neighbor[end] = [1, ] # norm to the first diag
-            (Nd >= 3) && (neighbor[end - 1] = [Nd - 2, ]) # last diag to the second last, possible only for Nd>=3
+            neighbor = Vector{Vector{Int}}([[d - 1, d + 1] for d = 1:Nd])
```
Comment (iintSjds) [inline src/variable.jl:133]: what's the difference between 
for d in a:b
and
for d = a:b

Comment (kunyuan) [reply src/variable.jl:133]: no difference

File: src/variable.jl (hunk 3 above)
Comment (kunyuan) [reply src/variable.jl:240]: There is no access to the user-defined  rng in this function.

**Pull request merged**